.github/workflows/ai-tools.yaml

```
@@ -0,0 +1,39 @@
+name: AI Assistant
+
+on:
+  issue_comment:
+    types: [created]
+  pull_request_review_comment:
+    types: [created]
+  issues:
+    types: [opened, assigned]
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  claude-code-action:
+    if: |
+      (github.event_name == 'issue_comment' && contains(github.event.comment.body, '@tt-ai')) ||
+      (github.event_name == 'pull_request_review_comment' && contains(github.event.comment.body, '@tt-ai')) ||
+      (github.event_name == 'pull_request_review' && contains(github.event.review.body, '@tt-ai')) ||
+      (github.event_name == 'issues' && contains(github.event.issue.body, '@tt-ai'))
+    runs-on: [arch-wormhole_b0, in-service, n300]
+    permissions:
+      contents: read
+      pull-requests: read
+      issues: read
+      id-token: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
```

If permitting a build, we should have more history so that the version is computed properly.  I don't recall whether we fail for an invalid version.  But certainly we don't check for it, so it could fail in the future someday.  Adding some depth means we shouldn't run into that now or tomorrow.

---

We use 500 in the official builds.  That would be a reasonable number, I think.

```suggestion
          fetch-depth: 500
```